stdlib/exit.c

```diff
@@ -137,4 +165,4 @@ exit (int status)
 {
   __run_exit_handlers (status, &__exit_funcs, true, true);
 }
-libc_hidden_def (exit)
+libc_hidden_def (exit)
```

Comment:
theres a lot of files where we remove the newlines at the end, pedantic but should clean that up